src/plugins/mer/meremulatordevicewidget.cpp

```diff
@@ -105,16 +126,23 @@ void MerEmulatorDeviceWidget::updatePortInputsEnabledState()
     QString message = tr("Stop emulator to unlock this field for editing.");
     if (!isStored)
         message = tr("Finish adding the device to unlock this field for editing.");
+    const QString toolTip = QString("<font color=\"red\">%1</font>");
 
-    m_ui->qmlLivePortsInfoLabel->setToolTip(QString("<font color=\"red\">%1</font>").arg(message));
-    m_ui->sshPortInfoLabel->setToolTip(QString("<font color=\"red\">%1</font>").arg(message));
+    m_ui->qmlLivePortsInfoLabel->setToolTip(toolTip.arg(message));
+    m_ui->sshPortInfoLabel->setToolTip(toolTip.arg(message));
+    m_ui->deviceModelInfoLabel->setToolTip(toolTip.arg(message));
 
     bool canEdit = vmOff && isStored;
 
     m_ui->sshPortSpinBox->setEnabled(canEdit);
     m_ui->sshPortInfoLabel->setVisible(!canEdit);
     m_ui->qmlLivePortsLineEdit->setEnabled(canEdit);
     m_ui->qmlLivePortsInfoLabel->setVisible(!canEdit);
+    m_ui->deviceModelComboBox->setEnabled(canEdit);
+    m_ui->deviceModelInfoLabel->setVisible(!canEdit);
+
+    if (canEdit)
+        device->setDeviceModel(device->deviceModel()); // In case user changed profile when emulator is running
```

Comment:
What does it do more exactly? It seems suspicious to me since the controls are disabled when emulator is running.

Reply:
This adds "*" to name if user changed profile settings (resolution for example). So "Device (400x600)" become "Device (400x600) *" if profile was edited while VM is running.